### PR TITLE
Integrate tailwind with trunk properly

### LINKED ISSUE #371: Enhancement: Use built in support for tailwind in trunk

### PR DESCRIPTION
**Pull request by n1tranquilla**
Closes #371

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -14,28 +14,19 @@ default:
 
 set dotenv-load := true
 
-tailwind:
```
Comment (n1tranquilla) [inline Justfile:17]: We no longer need to explicitly invoke tailwind. `trunk build` will invoke.

File: Justfile [hunk 2]
```diff
@@ -14,28 +14,19 @@ default:
...
-  npx tailwindcss -i assets/css/input.css -o assets/css/styles.css --minify
-
-tailwind-watch:
```
Comment (n1tranquilla) [inline Justfile:20]: We no longer need to explicitly watch for file changes. `trunk serve` will watch for changes in files and re-serve.

File: Justfile [hunk 3]
```diff
@@ -14,28 +14,19 @@ default:
...
+
+build: build_npm
   trunk build
 
-build-release:
-  npm install
-  just tailwind
-  cargo build --release
+build-release: build_npm
+  trunk build --release
```
Comment (n1tranquilla) [inline Justfile:24]: `trunk` handles building of webroot. `cargo` not needed for this.

File: Justfile [hunk 4]
```diff
@@ -14,28 +14,19 @@ default:
...
 
 clean:
   rm -rf dist
   trunk clean
   cargo clean
-  rm -f assets/css/styles.css
```
Comment (n1tranquilla) [inline Justfile:38]: No need to cleanup css files now.

File: Justfile [hunk 5]
```diff
@@ -68,16 +59,11 @@ format:
...
-dev: build
-  node scripts/wait-on-port.js trunk serve --port="{{trunk_port}}" --open -- "{{trunk_port}}" -- just tailwind-watch
+dev: build_npm
+  trunk serve --port="{{trunk_port}}" --open
 
-release: build-release
+publish: 
   trunk build --release --filehash true
-
-pre_build:
```
Comment (n1tranquilla) [inline Justfile:77]: We no longer need this prebuild hook for tailwind

**Comment by jhult** [comment]
Squash on merge

**Pull request merged**